src/models/model.ts

```diff
@@ -206,6 +206,7 @@ class Model {
             return resp.json();
         }).then((userKeybindings) => {
             this.keybindManager.setUserKeybindings(userKeybindings);
+            console.log("all keybindings:", this.keybindManager.getAllKeybindUIDescriptions());
```

Comment:
remove console.log.  we can see this by just accessing it directly in the console if we want it. `GlobalModel.keybindManager.getAllKeybindUIDescriptions()`